### ISSUE TITLE
NTR: Cervical motion tenderness (uterine) (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7964]
New term request Cervical motion tenderness (uterine) (seidel_190301171845): CMT; pain with movement of cervix on bimanual exam; uterus  UMLS: 0238953

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000107
- label: Cervical motion tenderness
- parents: [Pain (HP:0012531)](https://hpo.jax.org/app/browse/term/HP:0012531)
- pmid: PMID:31424870
- def: Pain that is elicited by moving the uterine cervix. This finding can be ascertained by pelvic exam. Cervical motion tenderness is a gynecological exam finding that could be indicative of peritoneal infection.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7964
- synonyms: Chandelier sign(NONE)
